### PR TITLE
Finalizing work on entering time

### PR DESCRIPTION
**Pull request by 7ep**

### CODE REVIEW

**Review by 7ep — COMMENTED** [review]
Reviewed the work.  Seems fine to me.

**Comment by 7ep** [comment]
User story checklist:

The following is used during story development as a reminder to the development team
what is needed for high-quality software.

functional development and considerations
    documented thoughtfully
        [ ] classes
        [ ] methods
        [ ] tests
        [ ] unusual aspects documented within code
        [ ] README's
        [ ] Developer documentation
        [ ] User documentation
        [ ] Log entries added
    correctness
        [ ] unit tests written
        [ ] Were the tests thorough, or only superficial?
        [ ] invariants applied (e.g. assert val > 10)
        [ ] integration tests written

non-functionals:
    [ ] perf
    [ ] security
    [ ] accessibility
    [ ] logging
    [ ] graceful degradation
    [ ] mobile-first

[ ] static analysis considered
[ ] risks considered
[ ] should it be refactored?

**Comment by 7ep** [comment]
@coveros-matt-taylor I would like to do a pair-walkthrough review on this code whenever you get a chance.

File: build.gradle [hunk 1]
```diff
@@ -91,16 +81,33 @@ javadoc {
     options.overview = "src/main/javadoc/overview.html"
 }
 
-// configuration for when we run the program as an application from the command line
-application {
```
Comment (coveros-matt-taylor) [inline build.gradle:95]: We may want to copy in some things from demo for debugging configuration

**Review by coveros-matt-taylor — APPROVED** [review]
These changes are excellent, though we should probably merge more frequently in the future.

**Pull request merged**